OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -96,6 +96,7 @@ public class UpdatePath
 				new ReplaceSequenceEmbeddedPalette(),
 				new UnhardcodeBaseBuilderBotModule(),
 				new UnhardcodeVeteranProductionIconOverlay(),
+				new RepairableBuilding(),
```

Comment:
Can we please give this a more descriptive name?

Reply:
Ah yes, when I wrote it I didn't know how update rules worked exactly. Wanted to fix this in the last force push but forgot